src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -5,101 +5,93 @@ excerpt: "xk6-browser brings browser automation and end-to-end testing to k6 whi
+| debug             | boolean  | `false` | All CDP messages and internal fine grained logs will be logged if set to `true`.                                                                                                                                                                                      |
+| devtools          | boolean  | `false` | Open up developer tools in the browser by default.                                                                                                                                                                                                                    |
+| env               | string[] | `null`  | Environment variables to set before launching browser process.                                                                                                                                                                                                        |
+| executablePath    | string   | `null`  | Override search for browser executable in favor of specified absolute path.                                                                                                                                                                                           |
+| headless          | boolean  | `true`  | Show browser GUI or not.                                                                                                                                                                                                                                              |
+| ignoreDefaultArgs | string[] | `null`  | Ignore any of the default arguments included when launching browser process.                                                                                                                                                                                          |
+| proxy             | string   | `null`  | Specify to set browser's proxy config.                                                                                                                                                                                                                                |
+| slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
+| timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
```

Comment:
@imiric or @inancgumus might know why that is?

Reply:
For PW compatibility, it might be an integer. For ease of use and Go compatibility, it might stay as string 🤔🤦 WDYT?

Reply:
In k6, we generally use these format strings for duration values, since it's friendlier to specify the unit along with the time, instead of wondering how integers will be interpreted.

So in this case we might want to deviate from PW/Puppeteer and adopt string durations in all cases. We actually have [an issue with how integers are parsed in some cases](https://github.com/grafana/xk6-browser/issues/456), and strings would avoid that ambiguity. This transition would take some time, though, which is why this is not consistent yet.